src/Workleap.OpenApi.MSBuild/CodeFirstProcess.cs

```diff
@@ -34,19 +36,24 @@ internal enum CodeFirstMode
         CodeFirstMode mode,
         CancellationToken cancellationToken)
     {
+        this._loggerWrapper.LogMessage("Installing dependencies...");
         await this.InstallDependencies(mode, cancellationToken);
         
+        this._loggerWrapper.LogMessage("Running Swagger...");
         var generateOpenApiDocsPath = (await this._swaggerManager.RunSwaggerAsync(openApiSwaggerDocumentNames, cancellationToken)).ToList();
 
         if (mode == CodeFirstMode.SpecGeneration)
         {
+            this._loggerWrapper.LogMessage("Updating specification files...");
             await this._specGeneratorManager.UpdateSpecificationFilesAsync(openApiSpecificationFilesPath, generateOpenApiDocsPath, cancellationToken);
```

Comment:
[nit]: would this considered Generation? the mode is `SpecGeneration` but the log/function refers to `Update`